deployment/constructs/lambda.ts

```diff
@@ -102,7 +102,10 @@ export class Lambda extends Construct {
         method,
         new apigateway.LambdaIntegration(this.lambda),
         {
-          authorizationType: apigateway.AuthorizationType.IAM,
+          authorizationType:
+            process.env.CDK_DEFAULT_ACCOUNT !== "000000000000"
+              ? apigateway.AuthorizationType.IAM
+              : undefined,
```

Comment:
```suggestion
          ...(process.env.CDK_DEFAULT_ACCOUNT !== "000000000000" && { authorizationType: apigateway.AuthorizationType.IAM })
```

I recently discovered this sort of thing is possible to conditionally add keys. Do you like it or think it's terrible? Part of me likes it but part of me thinks it isn't that readable.

Reply:
Yeah, I think that's less readable.  I'm flexible though.

Reply:
Cool cool 👍